pyqwikswitch/qwikswitch.py

```diff
@@ -231,9 +232,42 @@ def decode_imod(packet, channel=1):
             return None
     return None
 
+# byte 0:  0f = pir
+# byte 1:  firmware
+# byte 2 and 3:  number of seconds (in hex) that the PIR sends until a device should react.
 
+def decode_pir(packet):
+    """Decode a PIR"""
+    val = str(packet.get(QSDATA, ''))
+    if len(val) == 8 and val.startswith('0f'):
+        return int(val[-4:],16) > 0
+    return None
+
+# byte 0:  34 = temperature / humidity
+# byte 1:  firmware
+# byte 2-3:  humidity
+# byte 4-5:  temperature
+    
+def decode_temperature(packet):
+    """Decode the temperature"""
+    if len(val) == 12 and val.startswith('34'):
+      temperature = int(val[-4:], 16)
+      return round(float((-46.85 + (175.72 * (temperature / pow(2, 16))))))
+    return None
+
+def decode_humidity(packet):
+    """Decode the humidity"""
+    val = str(packet.get(QSDATA, ''))
+    if len(val) == 12 and val.startswith('34'):
+      humidity = int(val[4:-4],16)
+      return round(float(-6 + (125 * (humidity / pow(2, 16)))))
+    return None
+    
 SENSORS = {
     'imod': (decode_imod, bool),
     'door': (decode_door, bool),
+    'pir' : (decode_pir, bool),
+    'temperature' : (decode_temperature, int),
```

Comment:
Maybe this should be 'degC' instead of `int`?

In HA, the second parameter is either a [string unit](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/qwikswitch.py#L41-L43) or [bool type](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/qwikswitch.py#L154-L158) which becomes a binary_sensor

Your current unit will show `int:1`